library/patient.inc

```diff
@@ -895,13 +904,16 @@ function getPatientPhone($phone = "%", $given = "pid, id, lname, fname, mname, p
 
 function getPatientIds($given = "pid, id, lname, fname, mname", $orderby = "id ASC", $limit = "all", $start = "0")
 {
+    $sqlBindArray = array();
+
     $sql="select $given from patient_data order by $orderby";
 
     if ($limit != "all") {
-        $sql .= " limit $start, $limit";
+        $sql .= " limit ?, ?";
```

Comment:
For all of these limits escaping in this script, see my post above for how to do this correctly.